### PR TITLE
Fixes for examples

### PR DESCRIPTION
**Pull request by strickvl**
Various docs / readme / code fixes discovered while testing examples.

Testing of examples aren't yet complete, but can be merged in whenever makes sense.

## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/mlops-stacks/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: examples/airflow_orchestration/README.md [hunk 1]
```diff
@@ -68,16 +71,15 @@ This will bootstrap Airflow, start up all the necessary components and run them
 in the background. When the setup is finished, it will print username and
 password for the Airflow webserver to the console.
 
-{% hint style="warning" %} If you can't find the password on the console, you
+WARNING: If you can't find the password on the console, you
```
Comment (fa9r) [inline examples/airflow_orchestration/README.md:74]: Is there a reason why the hint style was removed?

**Review by fa9r — APPROVED** [review]
LGTM!

File: examples/airflow_orchestration/README.md (hunk 1 above)
Comment (strickvl) [reply examples/airflow_orchestration/README.md:74]: Yes. It doesn't render anywhere. Just shows up as plain text garbage.

**Review by htahir1 — APPROVED** [review]
Lets merge this asap

**Pull request merged**